packages/react-core/src/components/Checkbox/examples/CheckboxLabelWraps.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+
+export const CheckboxLabelWraps: React.FunctionComponent = () => (
+  <Checkbox
+    isLabelWrapped
+    label="Label wraps input example"
+    id="checkbox-label-wraps-input"
```

Comment:
We can omit the id in this example

Reply:
Looks like `id` is a required prop on the Checkbox (and on Radio as well). Is there any good reason for that, or can I make that prop optional?